config/connect-scalyr-sink.properties

```diff
@@ -16,8 +16,13 @@ api_key=<log write api key from https://app.scalyr.com/keys>
 # Optional enrichment attributes in key=value format which are added to the Scalyr event attributes
 #event_enrichment=key1=value1,key2=value2
 
-# Custom application event mapping - only needed for custom applications
-#custom_app_event_mapping=[{"matcher": {"attribute": "app.name", "value": "myapp"}, "eventMapping": {"message": "message", "logfile": "log.path", "serverHost": "host.hostname", "parser": "fields.parser", "version": "app.version", "appField1", "appField1", "appField2", "nested.appField2"}}]
+# Custom application event mapping - only needed for custom applications.  Regex is supported for the `matcher.value`.
+#custom_app_event_mapping=[{"matcher": {"attribute": "app.name", "value": "myapp.*"}, "eventMapping": {"message": "message", "logfile": "log.path", "serverHost": "host.hostname", "parser": "fields.parser", "version": "app.version", "appField1", "appField1", "appField2", "nested.appField2"}}]
+
+# Custom application event mapping with match all and send_entire_record=true
+# logfile, serverHost, and parser event mappings are still recommended, since these are special fields.
+# send_entire_record=true will send all other fields.
```

Comment:
This is a little confusing because it looks like you are trying to allow the customer to uncomment this line to turn on `send_entire_record`.  Now, you have a line down below that is set up to do that, but it is not clear they should uncomment that given your instructions here.

I would recommend that you remove this line (`# send_entire_record=true will send all other fields`) and add `#send_entire_record=true` after the customer app line.  That way they just need to uncomment those two lines to enable the described behavior.

I'm not in love with my approach but I think it is an improvement.